docs/onboarding/onboarding-feature.md

```diff
@@ -0,0 +1,173 @@
+---
+id: onboarding-feature
+title: Onboarding feature
+slug: /onboarding
+---
+
+# Introduction
+The onboarding feature enables experimentation with the 'new user onboarding flow'. The onboarding flow is presented to the user on each new install and is made up of a series of full screen 'views', referred to as '**cards**'. The purpose of the onboarding flow is to enable the user to configure a small number of app enhancing settings. Each card provides context for each setting and the ability to enable/skip.
```

Comment:
This is for fenix only, I think?

It would be amazing if Roux could see this.

Reply:
Have updated the doc to show as Android only, will ping Roux 👍

Reply:
🚀

Reply:
I'll spend some time next week creating documentation for iOS as well!